File: src/Lession2/Ex9.java
package Lession2;

import java.util.Scanner;

public class Ex9 {

    static void main() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter N of Fibbonacci seq: ");
        int N = sc.nextInt();
        System.out.println(fibbonacci(N));
    }

    private static int fibbonacci (int n) {
        if (n == 0) {
            return 0;
        } else if (n == 1) {
            return 1;
        } else {
            return fibbonacci(n - 1) + fibbonacci(n - 2);
        }
    }
}

File: src/Lession2/Ex9.kt
package Lession2

fun main() {
    print("Enter N of Fibbonacci seq: ")
    val n = readln().toInt();

    val nf = fibbonacci(n)
    println("$nf")
}

fun fibbonacci(n: Int): Int {
    if (n == 0) {
        return 0
    } else if (n == 1) {
        return 1
    } else {
        return fibbonacci(n - 1) + fibbonacci(n - 2)
    }
}
